src/main/java/net/glowstone/util/library/Library.java

```diff
@@ -157,8 +157,7 @@ public static Library fromConfigMap(Map<?, ?> configMap) {
      * @return A map that is able to be serialized into a config.
      */
     public Map<?, ?> toConfigMap() {
-        // Using LinkedHashMap to keep the props in order when written into the config file.
-        Map<String, Object> configMap = new LinkedHashMap<>();
+        Map<String, Object> configMap = new HashMap<>();
```

Comment:
This isn't called within Glowstone (yet?), but the interface guarantees ordering for a config file.

Reply:
Yeah. Our tool finds that the order is actually not used because the map object is not traversed. Of course, you can maintain it as a LinkedHashMap if you want to assure the extensibility of the program. For the current version, the HashMap is a better choice.